src/top/dashboard.rs

```diff
@@ -23,6 +23,15 @@ trait ThousandsFormatter {
     fn thousands_format(&self) -> String;
 }
 
+impl ThousandsFormatter for u32 {
```

Comment:
Is this change related? Or did you just notice its absence? It seems like we could maybe macro-ize this over all integral types.

Reply:
This was actually an attempt to standardise the inputs of query/subscription functions in the API client, with GraphQL resolvers in the API server.

We generally take unsigned int types (and additionally use validation) on public-facing fields to make the input interface clearer, so I thought it might be worth doing the same internally to prevent negative inputs at the type level.

This turned out to be a little redundant, since the GraphQL spec (by virtue of using JSON as its transport, I guess), has no concept of unsigned integers, being typically interpreted by a JS server as a double-precision 64-bit binary format IEEE 754, so the unsigned types are just something async-graphql offers as a serialisation guard.

So, all values sent by graphql-client are actually `i64`s; we could pass `u32` around in our top/tap client code (which this was a starting point for), but we'd still be casting as the signed variants prior to the call to the API.

Which may be completely okay, but it was an effort I felt I started in vain/out of scope, and this is a remnant of it. I think it can be pulled out for now and possibly tackled later with more enthusiastic buy-in -- possibly in the form of a macro as you suggested for this particular trait.

Reply:
Ah, gotcha, I see the other update to `top` options. I'm happy to leave this in.